docs/navigate.rst

```diff
@@ -10,11 +10,11 @@ View VLAN table
 
    In the :guilabel:`Table` list, the VLAN table is selected by default.
 
-2. From the :guilabel:`Partition` list, select the partition you’re interested in.
+2. From the :guilabel:`Partition` list, select the partition you’re interested in. The **All partition** partition is supported for the VLAN table in v2.9 and later.
```

Comment:
The **All partitions** option is supported for the VLAN table in v2.9 and later.